crates/cli/src/bin/wasm-bindgen-test-runner/node.rs

```diff
@@ -139,8 +137,7 @@ pub fn execute(
             .env("NODE_PATH", env::join_paths(&path).unwrap())
             .arg("--expose-gc")
             .args(&extra_node_args)
-            .arg(&js_path)
-            .args(args),
```

Comment:
I assume at some point this was used to be able to pass arguments into Node.js/Deno. But this was already not functional because when `wasm-bindgen-test-runner` started parsing arguments in JS it would fail during runtime.